src/solver/conv_hip_implicit_gemm_3d_grouped_wrw_xdlops.cpp

```diff
@@ -309,6 +309,9 @@ struct CKArgs
     bool IsSupportedBy(const ConvPtr& conv_ptr) const
     {
         auto arg_ptr = MakeArgPtr(conv_ptr, nullptr, nullptr, nullptr, 1.0f, 0.0f);
+        // Creat dummy workspace to pass the ck IsSupportedArgument check.
+        int dummy_var = 1;
```

Comment:
I haven't looked int CK implementation and I'm curious about few questions:
Is it critical to pass `1`?
Can it be `0`?
Is it workspace size?

Reply:
It really does not matter what value we assign. [CK just needed a pointer](https://github.com/ROCm/composable_kernel/blob/develop/include/ck/tensor_operation/gpu/device/impl/device_grouped_conv_bwd_weight_two_stage_xdl_cshuffle.hpp#L1338) not to be null.

Reply:
@CAHEK7 yes. CK just need a not-null pointer to check. It does matter what value we give here.